packages/es-ingest/src/main/java/software/uncharted/terarium/esingest/util/TimeFormatter.java

```diff
@@ -0,0 +1,46 @@
+package software.uncharted.terarium.esingest.util;
+
+import java.util.concurrent.TimeUnit;
+
+public class TimeFormatter {
```

Comment:
I believe that the java `Duration` class will do this for you, then you can just pass it a format (DD HH:mm:ss.SSS) to print in

Reply:
Good to know!